Use candidates dir

double16 opened the pull request.

double16 committed:
Fix bug where tool will not install during same run as sdkman install

double16 committed:
Fix package install exec command

double16 committed:
Fix package install exec command

double16 committed:
Do install using bash

willmoorefyi commented:
thanks for the commits @double16 !  Can you provide a bit of context as to why you want to do things this way?  It's been a while since I've used boxen but some of these changes seem a bit unconventional.

Thanks!

double16 commented:
So sorry I haven't responded to this! I found your comment while cleaning up my email.

I moved most of the code into the sdkman::package define because it reduces duplication. This is a common pattern in Puppet.

In addition, it's wasn't possible to install sdkman and the various packages (groovy, java, etc.) during the first run because the code assumed sdkman was initialized in the user's profile. Added the call to sdkman-init before each exec allows the first run to install sdkman and install packages.

You'll want to change metadata.json to your own name and repo too.

Thanks!
